docs/SupportedCommands/Commands.md

```diff
@@ -0,0 +1,70 @@
+## Supported Commands
```

Comment:
We should probably state in which version of the Engine each command was first supported.